### ISSUE TITLE
Report declaration/interface errors on start

### ISSUE THREAD

**Issue by CrabDude** [facebook/flow#89]
When an interface file contains an error, it fails silently, resulting in unexpected errors:
```javascript
declare module http {
  declare function createServer(callback?: (req: httpIncomingMessage, res: httpServerResponse) => void): httpServer
}

declare class httpServer {}
declare class httpIncomingMessage {}

// INVALID class method declaration syntax
declare class httpServerResponse {
  declare function end(chunk?: string): void
}
```

Results in:
```
http
Required module not found
```

**Comment by samwgoldman** [comment]
@bhosmer took care of this in 729c2aff651b049e749875ab3d13de6041bd708b. Now, your example results in:

```
flow-test/lib/test.js:10:11,18: Library parse error:
flow-test/lib/test.js:10:11,18: Unexpected identifier
```

which is much nicer!

**Comment by samwgoldman** [comment]
Fixed in 0.13
